### PR TITLE
Added GalaxyMine.de

### PR DESCRIPTION
**Pull request by TrismoTV**
## Type of change

- [ x] Add new directory for a server
- [ ] Update directory of a server
- [ ] Remove directory of a server
- [ ] Documentation Update

## Checklist

- [ x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [ x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x ] I have created the manifest.json with the required values

### CODE REVIEW

**Review by TrismoTV — COMMENTED** [review]
...

File: minecraft_servers/galaxymine/manifest.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+  "server_name": "galaxymine",
+  "nice_name": "GALAXYMINE",
+  "direct_ip": "galaxymine.de",
+  "social": {
+    "web": "https://galaxymine.de",
```
Comment (DoJapHD) [inline minecraft_servers/galaxymine/manifest.json:6]: Just a default Apache2 page

File: minecraft_servers/galaxymine/manifest.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    "web_shop": "https://shop.galaxymine.de",
```
Comment (DoJapHD) [inline minecraft_servers/galaxymine/manifest.json:7]: Not reachable

**Review by DoJapHD — CHANGES_REQUESTED** [review]

**Comment by DoJapHD** [comment]
Server is in maintenance mode;
Reopen PR once server is publicly available

**Comment by TrismoTV** [comment]
Please reopen the PR.

**Review by TrismoTV — COMMENTED** [review]
.

**Pull request closed without merging**